dist/index.html

```diff
@@ -56,37 +56,27 @@ <h2>Dinner</h2>
       <!-- FILTER BOXES -->
      <section class="selection-boxes">
       <div id="filter-input-wrapper">
-       <h2>Small Plates</h2>
+       <h2>Starters</h2>
        <div><input type="checkbox" class="checkbox" id="antipasti">
          <label for="antipasti">Antipasti</label></div>
        <div><input type="checkbox" class="checkbox" id="antipasto">
        <label for="antipasto">Antipasto</label></div>
        <div><input type="checkbox" class="checkbox" id="starter">
        <label for="starter">Starter</label></div>
-       <div><input type="checkbox" class="checkbox" id="snack">
-       <label for="snack">Snack</label></div>
        <div><input type="checkbox" class="checkbox" id="appetizer">
        <label for="appetizer">Appetizer</label></div>
-       <div><input type="checkbox" class="checkbox" id="hordoeuvre">
-       <label for="hordoeuvre">Hor D'oeuvre</label></div>
        <h2>Meals</h2>
        <div><input type="checkbox" class="checkbox" id="breakfast">
        <label for="breakfast">Breakfast</label></div>
-       <div><input type="checkbox" class="checkbox" id="morningmeal">
-       <label for="morningmeal">Morning Meal</label></div>
        <div><input type="checkbox" class="checkbox" id="brunch">
        <label for="brunch">Brunch</label></div>
        <div><input type="checkbox" class="checkbox" id="lunch">
        <label for="lunch">Lunch</label></div>
        <div><input type="checkbox" class="checkbox" id="dinner">
        <label for="dinner">Dinner</label></div>
-       <h2>Type</h2>
-       <div><input type="checkbox" class="checkbox" id="maincourse">
-       <label for="maincourse">Main Course</label></div>
-       <div><input type="checkbox" class="checkbox" id="maindish">
-       <label for="maindish">Main Dish</label></div>
-       <div><input type="checkbox" class="checkbox" id="sidedish">
-       <label for="sidedish">Side Dish</label></div>
```

Comment:
the multiple word tags were hard to implement so i took them out, they were pretty redundant anyways